docs/admin/admin.rst

```diff
@@ -28,7 +28,7 @@ and other settings:
 In the :guilabel:`Reports` section, you can check the status of your site, tweak
 it for :ref:`production`, or manage SSH keys used to access :ref:`vcs-repos`.
 
-Manage database objects in any of the sections below.
+Manage database objects in any of the sections above.
```

Comment:
I think the original meaning here was that these sections are below "Reports" section. I see the screenshot here is broken for past few releases, in 4.0 it was still correct (I will look into this issue):

![](https://docs.weblate.org/en/weblate-4.0/_images/admin.png)

Reply:
@comradekingu and I discussed this, but opted to patch it differently. No problem. Considering that "below" is ambiguous (as it could refer to sections ahead in the same documentation), what do you think it would be better? `Manage database objects under any of the sections.` ? Kingu suggested "underneath".

Reply:
That sounds fine to me. The key is that it is describing position on the page, not in the docs.

Reply:
@nijel I force-pushed the commits using "under" instead of the previous "above".